### PR TITLE
Fix casing #192

### PR DESCRIPTION
**Pull request by ricardoom**
Fixes the all capitalization issue(s) outlined in #192

### CODE REVIEW

**Comment by seanredmond** [comment]
Looking at the changed files, would what's in this PR affect the header as well?

**Comment by ricardoom** [comment]
I think all the classes are "pre-fixed" w/ `.nypl-some-class button { ... }` so i think it should not adversely affect the header. Locally I'm not seeing any changes there, but confirmation from a trusted 2nd and/or 3rd source would be great.

File: sass/_forms.scss [hunk 1]
```diff
@@ -565,7 +565,7 @@ $form-padding: 0.5rem;
     font-weight: 400;
 
     span {
```
Comment (EdwinGuzman) [inline sass/_forms.scss]: You can delete this whole rule.

**Comment by ricardoom** [comment]
@seanredmond unless you have some other issues w/ this PR, I'm going to merge... let me know what you think.

**Pull request merged**